plugin-server/src/utils/db/db.ts

```diff
@@ -1358,15 +1353,11 @@ export class DB {
         return hash
     }
 
-    // PluginLogEntry
+    // PluginLogEntry (stored in ClickHouse table `plugin_log_entries`)
```

Comment:
seems like a useless comment? I can infer this from the query below

we also don't specify these things anywhere else

Reply:
Just added this because normally those section titles are named after the respective Django model, but in this case the Django model just was made redundant and is no longer relevant (as we're all in on ClickHouse storage of this).